sonar-batch/src/main/java/org/sonar/batch/bootstrap/BatchPluginInstaller.java

```diff
@@ -31,6 +31,7 @@
 import org.apache.commons.io.FileUtils;
 import org.apache.commons.lang.CharUtils;
 import org.apache.commons.lang.StringUtils;
+import org.sonar.api.Plugin;
 import org.sonar.api.SonarPlugin;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.sonar.api.SonarPlugin'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AUselessImportCheck)